docs/articles/licensing/gplv3/gplv3-blacklist.rst

```diff
@@ -0,0 +1,86 @@
+GPLv3 licensed production software Vs automotive industry
+---------------------------------------------------------
+
+Modern cars have around 100 million lines of code running on them,12 with 70% 
```

Comment:
the number 12 is a superscript, so it should be aligned properly. Not sure if the problem is here or with the fromatting in git.

Reply:
@AshleyNair  Sure. Thank you. I will have a look.

Reply:
@AshleyNair test comment.

Reply:
@AshleyNair comment!

Reply:
@AshleyNair test comment!